packages/experimental/src/components/ExampleComponent/ExampleComponent.js

```diff
@@ -162,6 +169,11 @@ ExampleComponent.propTypes = {
    * The size for the buttons ('default', 'small' or 'field').
    */
   size: PropTypes.oneOf(['default', 'small', 'field']),
+
+  /**
+   * Optional style settings for the containing node.
+   */
+  style: PropTypes.object,
```

Comment:
Does this need a default for ...style to work?

Reply:
No. `...undefined` contributes nothing to an object literal, so `{...undefined} = {}` and `{...undefined, foo: 42} = {foo:42}`. As this is a useful way to handle unspecified props I've added commentary to clarify this in the ExampleComponent now.